CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## New Features
 
 ## Improvements
+- PR #1656: improve error message when using unsupported algo in FIL
+- PR #1644: Add `predict_proba()` for FIL binary classifier
```

Comment:
This change (and many other) are from a different pull request. Could you remove them from this pull request?